### ISSUE TITLE
[A11Y] Unable to select issue category/browser icon/screenshot upload using “Tab” key

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3396]
**Browser / Version**: Firefox Nightly 80.0a1 (2020-07-15)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Using "Enter" key, activate "Confirm" button and move to the next section.
5. Using "Tab" key, try to select a category.
6. Observe behavior.

**Expected Behavior:**
Category is active and can be selected.

**Actual Behavior:**
Category can't be selected.

**Notes:**
1. Screenshot attached.
2. The issue is also reproducible for browser selection and screenshot selection.
3. Taking this in consideration a report can't be submitted using keyboard keys.
![download](https://user-images.githubusercontent.com/12184325/87688573-cd7d1980-c78f-11ea-8dc7-09fc52edcff2.png)
4. According to "Inspector>Accessibility" tab, the elements "must be focusable and should have interactive semantics".
![image](https://user-images.githubusercontent.com/12184325/88666566-e9b27c00-d0e8-11ea-89ec-4c70949a2781.png)

Watchers:
@softvision-oana-arbuzov
@cipriansv

**Comment by miketaylr** [comment]
Let's maybe handle the screenshot stuff in https://github.com/webcompat/webcompat.com/issues/3419.

**Comment by softvision-oana-arbuzov** [comment]
I can confirm this is fixed.

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-16), Chrome 85.0.4183.102
Operating System: Windows 10 Pro
